decidim-core/config/locales/eu.yml

```diff
@@ -2,4 +2,4 @@ eu:
   locales:
     ca: Català
     en: English
-    es: Castellano
\ No newline at end of file
+    es: Castellano
```

Comment:
We should find a way to automatically find this and not needing to update every locale every time we add a new one :/